cli/build-public-library.js

```diff
@@ -70,6 +70,9 @@ function writeTSConfig() {
       'paths': {
         '@blackbaud/skyux-builder/*': [
           '*'
+        ],
+        '.skypageslocales/*': [
```

Comment:
Allow libraries to reference `.skypageslocales` during `skyux build-public-library`.